p2p/chain.py

```diff
@@ -313,9 +302,6 @@ class LightChainSyncer(BaseHeaderChainSyncer):
                           msg: protocol._DecodedMsgType) -> None:
         if isinstance(cmd, les.Announce):
             self._sync_requests.put_nowait(peer)
-        elif isinstance(cmd, les.BlockHeaders):
-            msg = cast(Dict[str, Any], msg)
-            self._handle_block_headers(tuple(cast(Tuple[BlockHeader, ...], msg['headers'])))
```

Comment:
Maybe it's worth leaving a comment here that `BlockHeaders` messages are not handled here because they're handled in a Peer method? Otherwise someone reading this code might have a hard time figuring out why those messages are not handled like the others

Reply:
Actually, we should probably leave the `elif` block here otherwise it will be handled in the `else` block, which logs a debug msg and may be confusing to someone reading the logs